tests/crypto/nist-p256/testcase.yaml

```
@@ -0,0 +1,16 @@
+common:
+  filter: dt_chosen_enabled("zephyr,entropy") and
+          CONFIG_CSPRNG_ENABLED and
+          CONFIG_HARDWARE_DEVICE_CS_GENERATOR and
+          not CONFIG_BUILD_WITH_TFM and
+          not (CONFIG_MBEDTLS and CONFIG_TINYCRYPT)
+  tags:
+    - mbedtls
+    - tinycrypt
+    - p256-m
+    - nist-p256
+tests:
+  crypto.nist-p256.mbedtls:
```

```suggestion
  crypto.secp256r1.mbedtls:
```

If we were to use the NIST definition, the curve-type would be named `NIST P-256` which is not as transferrable in terms of lower-case file naming etc.. I recommend using the SECG definition as this is already lower-case and have no added spaces or hyphens